NSS-Day-Cohort-42/rare-API-media-moguls
# Ap subscriptions

#### Changes Made
1. Modified `subscription.py` in `views` directory to successfully find a single active subscription.
​
#### Steps to Review
1. Checkout this branch locally.
```
git fetch --all
git checkout ap-subscriptions
```
2. Run the application.
```
npm start
```
3. Open a new Terminal tab (⌘T) and navigate to the server directory.
4. Initialize virtual environment, and run the server.
```
pipenv shell
python manage.py runserver
```
5. Test app functionality.
> Pull down client side ap-subscriptions branch
> On a nuked database, giving `http://localhost:8000/subscriptions/2/get_single_current_subscription` as a GET request in postman should give you a single open subscription, ID of 4
> On the client, navigate to `http://localhost:3000/users/2`
> Click "unsubscribe"
> Do the same request in Postman and a message should appear telling you that no subscription exists
6. View code file.
> Confirm file modifications are present as indicated above.
> Confirm no unused code or extraneous comments exist.
